decidim-initiatives/spec/controllers/decidim/initiatives/admin/initiatives_controller_spec.rb

```diff
@@ -5,14 +5,17 @@
 describe Decidim::Initiatives::Admin::InitiativesController, type: :controller do
   routes { Decidim::Initiatives::AdminEngine.routes }
 
-  let(:user) { create(:user, :confirmed, organization:) }
-  let(:admin_user) { create(:user, :admin, :confirmed, organization:) }
+  let(:user) { create(:user, :confirmed, admin_terms_accepted_at: Time.current, organization:) }
+  let(:admin_user) { create(:user, :admin, :confirmed, admin_terms_accepted_at: Time.current, organization:) }
```

Comment:
The `:admin` trait has already the admin terms accepted, so we don't need to explicitly define it here.
```suggestion
  let(:admin_user) { create(:user, :admin, :confirmed, organization:) }
```